python_ta/__init__.py

```diff
@@ -192,6 +190,15 @@ def _check(
                     load_default_config=load_default_config,
                 )
 
+                # Override error messages
```

Comment:
This code is good, but please move into the `reset_linter` function. This logically falls under "setting up the linter to be used for checking"